### PR TITLE
Feature/deployment tweaks

### PR DESCRIPTION
**Pull request by CloudNiner**

### CODE REVIEW

File: Vagrantfile [hunk 1]
```diff
@@ -111,7 +111,11 @@ Vagrant.configure(VAGRANTFILE_API_VERSION) do |config|
     if testing?
         app.vm.synced_folder ".", "/opt/app"
     else
-        app.vm.synced_folder ".", "/opt/app", :nfs => true, :mount_options => ['actimeo=2']
+        app.vm.synced_folder ".", "/opt/app", :nfs => true, :mount_options => [
+            ("nfsvers=3" if ENV.fetch("CAC_NFS_VERSION", 4).to_i == 3),
```
Comment (sharph) [inline Vagrantfile]: Did you fix your nfs troubles with this?

Comment (sharph) [inline Vagrantfile]: Also, might something like CAC_USE_NFS_3 be better here? Seems weird that you are specifying a boolean with what appears to be any arbitrary version.

**Comment by sharph** [comment]
+1

File: Vagrantfile (hunk 1 above)
Comment (CloudNiner) [inline Vagrantfile]: Yikes, that's what i get for copy-pasting. I'll update to what you suggest.

Fix is from here: https://github.com/azavea/nih-wayfinding/pull/52

**Pull request merged**